src/RouteServiceProvider.php

```diff
@@ -195,6 +197,32 @@ private function registerRouteMiddlewares()
         Route::aliasMiddleware('permission', Permission::class);
     }
 
+    /**
+     * Registers Route macros.
+     *
+     * @return void
+     */
+    protected function registerMacros()
```

Comment:
Ah yes I was still supposed to do this! Please add void as a return type, then we can remove it from the docblock

Reply:
Thank you! I made that change.